### PR TITLE
Add 西安理工大学 官网主页 教务处

### PR DESCRIPTION
**Pull request by mocusez**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```route
/xaut/index/tzgg
/xaut/index/mtbd
/xaut/index/xyyw
/xaut/index/xshd
/xaut/jwc/tzgg
/xaut/jwc/gkgs
/xaut/jwc/gzzd
/xaut/jwc/jggs
/xaut/jwc/jsjg
/xaut/jwc/jsxx
/xaut/jwc/xwdt
```

## 新RSS检查列表 / New RSS Script Checklist

- [x] New Route
- [x] Documentation
  - [x] CN
  - [ ] EN
- [ ] 全文获取 fulltext
  - [x] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] 日期和时间 date and time
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

## 说明 / Note

全文获取的版本通不过去，不清楚怎么回事

https://github.com/DIYgod/RSSHub/pull/8795

### CODE REVIEW

File: lib/v2/xaut/index.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    if (dic_title[category] === undefined) {
+        category = 'tzgg';
+    }
+
+    const response = await got({
+        method: 'get',
+        url: 'http://www.xaut.edu.cn/index/' + dic_html[category],
+        headers: {
+            accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8',
+            'user-agent': 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36',
```
Comment (TonyRL) [inline lib/v2/xaut/index.js:20]: UA should be set via env

https://github.com/DIYgod/RSSHub/blob/b5be05b1e2bdc18e6fec5ebe137cca9937175ff3/lib/config.js#L40

File: lib/v2/xaut/jwc.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    if (dic_title[category] === undefined) {
+        category = 'tzgg';
+    }
+
+    const response = await got({
+        method: 'get',
+        url: rootUrl + dic_html[category],
+        headers: {
+            accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8',
+            'user-agent': 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36',
```
Comment (TonyRL) [inline lib/v2/xaut/jwc.js:20]: UA should be set via env

https://github.com/DIYgod/RSSHub/blob/b5be05b1e2bdc18e6fec5ebe137cca9937175ff3/lib/config.js#L40

File: docs/university.md [hunk 3]
```diff
@@ -2103,6 +2103,33 @@ type 列表：
 
 <Route author="DylanXie123" example="/xjtu/ee/1114" path="/xjtu/job/:id?" :paramsDesc="['栏目id，默认请求`1114`，可在 URL 中找到']"  />
 
+## 西安理工大学
+
+### 学校主页
+
+<Route author="mocusez" example="/xaut/index/tzgg" path="/xaut/index/:category?" :paramsDesc="['通知类别,默认为通知公告']">
```
Comment (TonyRL) [inline docs/university.md:2110]:
```suggestion
<Route author="mocusez" example="/xaut/index/tzgg" path="/xaut/index/:category?" :paramsDesc="['通知类别，默认为通知公告']">
```

File: docs/university.md [hunk 4]
```diff
@@ -2103,6 +2103,33 @@ type 列表：
...
+
+| 通知公告 | 校园要闻 | 媒体播报 | 学术活动 |
+| :------: | :------: | :------: | :------: |
+|   tzgg   |   xyyw   |   mtbd   |   xshd   |
+
+</Route>
+
+### 教务处
+
+<Route author="mocusez" example="/xaut/jwc/tzgg" path="/xaut/jwc/:category?" :paramsDesc="['通知类别,默认为通知公告']">
```
Comment (TonyRL) [inline docs/university.md:2120]:
```suggestion
<Route author="mocusez" example="/xaut/jwc/tzgg" path="/xaut/jwc/:category?" :paramsDesc="['通知类别，默认为通知公告']">
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: docs/university.md (hunk 3 above)
Comment (TonyRL) [reply docs/university.md:2110]: Please use Chinese punctuation.

**Comment by mocusez** [comment]
???So, What‘s wrong? All the route I have been tested on my computer
:pensive: :pensive:

**Comment by TonyRL** [comment]
FYI, you have to put your new routes in 

\`\`\`routes
/somethingA...
/somethingB...
\`\`\`

not "route"

**Comment by mocusez** [comment]
Can you give me more suggestyion？I don't truly know what had happened.:sob:

**Comment by mocusez** [comment]
Ouch!!!  I understand, a stupid mistake

**Comment by TonyRL** [comment]
Apart from that, if you have fixed that problem and push some changes, it might reopen(I don't think so).

**Pull request closed without merging**